graph-db/serverless/src/utils/cmr/indexCmrCollection.js

```diff
@@ -65,6 +67,12 @@ export const indexCmrCollection = async (collection, gremlinConnection) => {
   const { value = {} } = dataset
   const { id: datasetId } = value
 
+  if (projects && projects.length > 0) {
```

Comment:
Is the wrapper and check for elements necessary or can forEachAsync handle an empty array without it?

Reply:
it will not work if projects is null without the check.